tracer/test/Datadog.Trace.Debugger.IntegrationTests/DebuggerManagerDynamicTests.cs

```diff
@@ -0,0 +1,301 @@
+// <copyright file="DebuggerManagerDynamicTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Debugger.IntegrationTests.Assertions;
+using Datadog.Trace.Debugger.IntegrationTests.Helpers;
+using Datadog.Trace.Debugger.Sink;
+using Datadog.Trace.TestHelpers;
+using Samples.Probes.TestRuns.SmokeTests;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+#nullable enable
+namespace Datadog.Trace.Debugger.IntegrationTests;
+
+#if !NETCOREAPP2_1
```

Comment:
This isn't a big deal, as we don't test on 2.1 anymore, but why do we need this explicit exclude?

Reply:
We have a crash in ClrMD on NETCOREAPP2_1